tests/models/univnet/test_feature_extraction_univnet.py

```diff
@@ -0,0 +1,345 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+import os
+import random
+import tempfile
+import unittest
+
+import numpy as np
+from datasets import Audio, load_dataset
+
+from transformers import UnivNetGanFeatureExtractor
+from transformers.testing_utils import check_json_file_has_correct_format, require_torch, slow
+from transformers.utils.import_utils import is_torch_available
+
+from ...test_sequence_feature_extraction_common import SequenceFeatureExtractionTestMixin
+
+
+if is_torch_available():
+    import torch
+
+
+global_rng = random.Random()
+
+
+# Copied from tests.models.whisper.test_feature_extraction_whisper.floats_list
+def floats_list(shape, scale=1.0, rng=None, name=None):
+    """Creates a random float32 tensor"""
+    if rng is None:
+        rng = global_rng
+
+    values = []
+    for batch_idx in range(shape[0]):
+        values.append([])
+        for _ in range(shape[1]):
+            values[-1].append(rng.random() * scale)
+
+    return values
+
+
+class UnivNetGanFeatureExtractionTester(unittest.TestCase):
+    def __init__(
+        self,
+        parent,
+        batch_size=7,
```

Comment:
Batch size 2 should be sufficient and considerably faster!
```suggestion
        batch_size=2,
```

Reply:
When switching the `batch_size` to `2`, I get some errors from tests that depend on [`SequenceFeatureExtractionTestMixin._check_padding`](https://github.com/huggingface/transformers/blob/2f3ea08a077ba3133fa8a604b22436cad250b055/tests/test_sequence_feature_extraction_common.py#L98) (e.g. `test_padding_from_list`, `test_truncation_from_array`). I'm not sure if there's an issue with `UnivNetGanFeatureExtractor` or if the test doesn't work for `batch_size=2`, will investigate further.

[It seems that most other audio feature extractors that use `SequenceFeatureExtractionTestMixin` seem to use a `batch_size` of `7`, such as [Whisper](https://github.com/huggingface/transformers/blob/2f3ea08a077ba3133fa8a604b22436cad250b055/tests/models/whisper/test_feature_extraction_whisper.py#L62), [Wav2Vec2](https://github.com/huggingface/transformers/blob/2f3ea08a077ba3133fa8a604b22436cad250b055/tests/models/wav2vec2/test_feature_extraction_wav2vec2.py#L50), and [SpeechT5](https://github.com/huggingface/transformers/blob/2f3ea08a077ba3133fa8a604b22436cad250b055/tests/models/speecht5/test_feature_extraction_speecht5.py#L56).]

Reply:
Okay fine to keep it then in that case! Thanks for checking

Reply:
As a sanity check, if I set `batch_size` to `2` for `WhisperFeatureExtractionTester`, the same 4 tests in `WhisperFeatureExtractionTest` (`test_padding_from_array`, `test_padding_from_list`, `test_truncation_from_array`, `test_truncation_from_list`) which depend on [`SequenceFeatureExtractionTestMixin._check_padding`](https://github.com/huggingface/transformers/blob/2f3ea08a077ba3133fa8a604b22436cad250b055/tests/test_sequence_feature_extraction_common.py#L98) will fail.